src/main/java/frc/robot/constants/SHOOTER.java

```diff
@@ -50,7 +50,7 @@ public enum RPM_SETPOINT {
     NONE(0),
 
     SLOW(600),
-    SPEAKER(4201),
+    SPEAKER(6000),
```

Comment:
Have we confirmed that 6000 is max?

Reply:
From last test, 6000RPM was max, but this was without gear ration change so we dont know if we need 6000RPM